environments/common/inventory/group_vars/builder/defaults.yml

```diff
@@ -6,3 +6,6 @@ openhpc_slurm_service_started: false
 nfs_client_mnt_state: present
 openhpc_rebuild_reconfigure: false
 update_enable: true
+block_devices_partition_state: skip
```

Comment:
Does this mean that we need to run ansible against any new node we enroll? Even if we boot from the packer created image?

Reply:
Well, if we're booting a new node with a packer-built image, that boot process can't create the partition. So the only sensible thing to do is skip, I think?

For cinder volumes that's fine as we can run the ansible first and then the packer-built image just has a directory in the right place and fstab configured to enable the volume to be mounted.

Reply:
I see; I guess the alternative would be to some sort of cloud-init script or similar to setup these block devices on first boot, but I'm not sure if there is a requirement to run the ansible against a new node already?